### PR TITLE
Docky-2.2.1.1

### PR DESCRIPTION
**Pull request by venv21**
http://wiki.go-docky.com

### CODE REVIEW

**Comment by Vaelatern** [comment]
Do everybody a favor and shrink the number of commits? Only should be New Package OR update commits.

**Comment by venv21** [comment]
Sorry, this was my first contribution. For me the easiest way seems to delete this pull request and open a new one with better commits, right?

**Comment by Vaelatern** [comment]
No

**Comment by Vaelatern** [comment]
The best way is to rebase. Please don't close and re-open PRs, it loses the history and makes it more work.

**Comment by pullmoll** [comment]
@venv21 get familiar with `git rebase -i <tag>` and the way you can reorder, squash or drop commits. When done just do a `git push -f` to the branch in your fork.

**Comment by venv21** [comment]
Learned something new, thank you!

**Comment by Vaelatern** [comment]
Ah, lots of packages. Sorry for the delay in reviewing.

File: srcpkgs/gio-sharp/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'gio-sharp'
+pkgname=gio-sharp
+version=0.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="libtool intltool pkg-config autoconf automake"
+makedepends="gtk-sharp-2 mono gnome-sharp glib-devel"
+depends="gtk-sharp-2 gnome-sharp"
+only_for_archs="i686 x86_64"
```
Comment (Vaelatern) [inline srcpkgs/gio-sharp/template]: Why?

File: srcpkgs/gio-sharp/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'gio-sharp'
+pkgname=gio-sharp
+version=0.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="libtool intltool pkg-config autoconf automake"
```
Comment (Vaelatern) [inline srcpkgs/gio-sharp/template]: `automake` pulls in `autoconf`, feel free to remove `autoconf`

**Comment by Gottox** [comment]
No process for 5 months

**Pull request closed without merging**